x-pack/packages/kbn-cloud-security-posture/common/constants.ts

```diff
@@ -23,7 +23,8 @@ export const STATUS_ROUTE_PATH = '/internal/cloud_security_posture/status';
 export const STATUS_API_CURRENT_VERSION = '1';
 
 /** The base path for all cloud security posture pages. */
-export const CLOUD_SECURITY_POSTURE_BASE_PATH = '/cloud_security_posture';
+export const CLOUD_SECURITY_POSTURE_PATH = 'cloud_security_posture';
```

Comment:
as far as I can see we don't use this anywhere anymore separately. Is this change still needed?